src/app/organizations/organization/organization.component.html

```diff
@@ -28,8 +28,8 @@
     </mat-card>
     <mat-card fxFlex class="my-3">
       <div fxLayout="row" fxLayout.lt-sm="column" fxLayoutGap="10px">
-        <div >
-          <img [src]="org.avatarUrl != null ? gravatarUrl : '../../../assets/images/dockstore/PlaceholderLC.png'" class="img-circle" height="150" width="150">
+        <div>
+          <img [src]="(gravatarUrl$ | async ) || '../../../assets/images/dockstore/PlaceholderLC.png'" class="img-circle" height="150" width="150">
```

Comment:
I'm not sure if this is legit, although it worked when @NatalieEO tried it. @garyluu , is this OK?

Reply:
I think it should.  From the pipe map, `(gravatarUrl$ | async)` looks like it will always return null or the actual gravatarUrl.  Beyond that, it's just business as usual